### PR TITLE
Added error ui and constants/errors.js

### PR DESCRIPTION
**Pull request by Ronan-Codes**
- Example Usage: <Error errorType={errorType.invalidPassword.errorText}/>

- import errorOptions in parent component and choose the proper errorType

- Look at comments in error/index.tsx for info about using redux to change state

### CODE REVIEW

File: src/stores/actions/error.action.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+export const CHANGE_TEXT = 'CHANGE_TEXT';
+export const CLEAR_TEXT = 'CLEAR_TEXT';
+
```
Comment (GoldenaArcher) [inline src/stores/actions/error.action.ts]: Updating redux is unnecessary, it's already handled

File: src/stores/reducers/error.reducer.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { ErrorType, CHANGE_TEXT, CLEAR_TEXT } from './../actions/error.action';
+
+export interface ErrorState {
+  errorText: string;
+}
+
```
Comment (GoldenaArcher) [inline src/stores/reducers/error.reducer.ts]: Updating redux is unnecessary, it's already handled

File: src/stores/reducers/index.ts [hunk 3]
```diff
@@ -1,13 +1,16 @@
 import { combineReducers } from '@reduxjs/toolkit';
 import authReducer, { AuthState } from './auth.reducer';
+import errorReducer, { ErrorState } from './error.reducer';
 
 export interface AppState {
```
Comment (GoldenaArcher) [inline src/stores/reducers/index.ts]: Updating redux is unnecessary, it's already handled

File: src/ui/error/index.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import { Alert } from '@mui/material';
+
+import { useSelector } from 'react-redux';
+// import { changeText, clearText } from '../../stores/actions/error.action';
+import { AppState } from '../../stores/reducers';
+import { ErrorState } from '../../stores/reducers/error.reducer';
```
Comment (GoldenaArcher) [inline src/ui/error/index.tsx]: My bad, I wasn't planning very well, please refer MUI's `Text field` for more information, changes should just be a couple lines

File: src/styles/index.scss [hunk 5]
```diff
@@ -4,3 +4,4 @@
 // components
 @import '../ui/header/header';
 @import '../ui/toolbar/toolbar';
+@import '../ui/error/error'
```
Comment (GoldenaArcher) [inline src/styles/index.scss]: Try to clean up empty/unused files.

File: src/stores/reducers/index.ts [hunk 6]
```diff
@@ -1,13 +1,16 @@
 import { combineReducers } from '@reduxjs/toolkit';
 import authReducer, { AuthState } from './auth.reducer';
```
Comment (GoldenaArcher) [inline src/stores/reducers/index.ts]: Updating redux is unnecessary, it's already handled

**Review by GoldenaArcher — CHANGES_REQUESTED** [review]

File: src/ui/error/index.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+import { Alert } from '@mui/material';
+
+// link to error constants
```
Comment (GoldenaArcher) [inline src/ui/error/index.tsx:4]: Integrate with Input from MUI later.

File: src/constants/errors.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+// pseudo error data
+let errorOptions = {
+  'lockedOneMin': 'Your account has been locked because of consecutive failed login attempts. Please try again in 30 minutes.',
+  'lockedThirtyMin': 'Your account has been locked because of consecutive failed login attempts. Please try again in 30 minutes.',
+  'invalidCredsOne': 'Invalid Credentials. Attempts Remaining: 1',
```
Comment (GoldenaArcher) [inline src/constants/errors.js]: Need re-formatter here.

**Review by GoldenaArcher — CHANGES_REQUESTED** [review]

File: src/ui/error/index.tsx (hunk 7 above)
Comment (Ronan-Codes) [reply src/ui/error/index.tsx:4]: Noted

**Pull request closed without merging**